packages/test/test-end-to-end-tests/src/test/gc/gcDatastoreDuplicateRoutes.spec.ts

```diff
@@ -57,4 +61,52 @@ describeNoCompat("GC Data Store Duplicates", (getTestObjectProvider) => {
         assert(gcObject.type === SummaryType.Handle, "Expected a handle!");
         assert(gcObject.handleType === SummaryType.Tree, "Expected a gc tree handle!");
     });
+
+    it("Back routes added by GC are removed when passed from data stores to DDSes", async () => {
+        const settings = {};
+        const configProvider = mockConfigProvider(settings);
+        const newConfig: ITestContainerConfig = {
+            runtimeOptions: defaultGCConfig.runtimeOptions,
+            loaderProps: { configProvider },
+        };
+        settings["Fluid.GarbageCollection.TrackGCState"] = "false";
```

Comment:
This test will stop working when TrackGCState config is removed. You can instead make sure that the GC data changes so that you get a tree. For instance, after loading summarizer2, you can create another data store and make it referenced. That will force GC tree to be generated.